### ISSUE TITLE
Custom error messages

### ISSUE THREAD

**Issue by programmervarunshridhar** [SKLn-Rad/Dart-Fluent-Validation#3]
ruleFor((project) => project.id).notNull().notEmpty().withMessage("id cannot be empty")

withMessage (WithMessage)  method is from the C# counterpart, but, could not find it in dart.

is it possible to provide a custom error message?

**Comment by the-best-is-best** [comment]
yes available but syntax ruleFor((project) => project.id).notNull(message: "your message").notEmpty(message:"your message");
